sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample2.1_AnalyzeSentimentWithOpinionMiningAsync.cs

```diff
@@ -33,33 +34,33 @@ public async Task AnalyzeSentimentWithOpinionMiningAsync()
                 "We changed rooms as the toilet smelled."
             };
 
-            AnalyzeSentimentResultCollection reviews = await client.AnalyzeSentimentBatchAsync(documents, options: new AnalyzeSentimentOptions() { AdditionalSentimentAnalyses = AdditionalSentimentAnalyses.OpinionMining });
+            AnalyzeSentimentResultCollection analyzedReviews = await client.AnalyzeSentimentBatchAsync(reviews, options: new AnalyzeSentimentOptions() { MineOpinions = true });
 
-            Dictionary<string, int> complaints = GetComplaint(reviews);
+            Dictionary<string, int> complaintTargets = GetNegativeOpinionTargets(analyzedReviews);
 
-            var negativeAspect = complaints.Aggregate((l, r) => l.Value > r.Value ? l : r).Key;
-            Console.WriteLine($"Alert! major complaint is *{negativeAspect}*");
+            var negativeOpinionTarget = complaintTargets.Aggregate((a, b) => a.Value > b.Value ? a : b).Key;
+            Console.WriteLine($"Alert! major complaint target is *{negativeOpinionTarget}*");
             Console.WriteLine();
-            Console.WriteLine("---All complaints:");
-            foreach (KeyValuePair<string, int> complaint in complaints)
+            Console.WriteLine("---All complaint targets:");
+            foreach (KeyValuePair<string, int> complaintTarget in complaintTargets)
             {
-                Console.WriteLine($"   {complaint.Key}, {complaint.Value}");
+                Console.WriteLine($"   OpinionTarget: {complaintTarget.Key}, Number of negative descriptions: {complaintTarget.Value}");
```

Comment:
Clarifying what these actually are ... it's the complaint target and the number of times it was called out as negative.